lib/Resizable.js

```diff
@@ -182,6 +186,27 @@ export default class Resizable extends React.Component<Props, State> {
       const canDragX = (this.props.axis === 'both' || this.props.axis === 'x') && ['n', 's'].indexOf(axis) === -1;
       const canDragY = (this.props.axis === 'both' || this.props.axis === 'y') && ['e', 'w'].indexOf(axis) === -1;
 
+      // Check if handle's position has moved since last resizeHandler call - adjust deltas accordingly
```

Comment:
It'd be great if you could expand upon this comment with some of the detail in this PR: that is, what happens, why it only affects `n` and `w` directions, and how this fixes it.

Reply:
Fleshed these comments out a good bit. Hopefully they're clear enough now 🤞